### PR TITLE
Added makefile fix for make out/minikube on windows

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,8 +47,11 @@ MKGOPATH := if [ ! -e $(GOPATH)/src/$(ORG) ]; then mkdir -p $(GOPATH)/src/$(ORG)
 LOCALKUBEFILES := go list  -f '{{join .Deps "\n"}}' ./cmd/localkube/ | grep k8s.io | xargs go list -f '{{ range $$file := .GoFiles }} {{$$.Dir}}/{{$$file}}{{"\n"}}{{end}}'
 MINIKUBEFILES := go list  -f '{{join .Deps "\n"}}' ./cmd/minikube/ | grep k8s.io | xargs go list -f '{{ range $$file := .GoFiles }} {{$$.Dir}}/{{$$file}}{{"\n"}}{{end}}'
 
-out/minikube: out/minikube-$(GOOS)-$(GOARCH)
-	cp $(BUILD_DIR)/minikube-$(GOOS)-$(GOARCH) $(BUILD_DIR)/minikube
+ifeq ($(GOOS),windows)
+	IS_EXE = ".exe"
+endif
+out/minikube: out/minikube-$(GOOS)-$(GOARCH)$(IS_EXE)
+	cp $(BUILD_DIR)/minikube-$(GOOS)-$(GOARCH)$(IS_EXE) $(BUILD_DIR)/minikube
```
Comment (jimmidyson) [inline Makefile]: Target file needs appending `$(IS_EXE)` too, i.e. `$(BUILD_DIR)/minikube$(IS_EXE)`.

Comment (aaron-prindle) [inline Makefile]: Good catch, thanks.  Done.

Comment (aaron-prindle) [inline Makefile]: Will this work properly though for rebuilding.  I am not sure how makefiles check for updates, will the makefile with target out/makefile look for out/makefile.exe?

**Comment by dlorenc** [comment]
> Will this work properly for rebuilding? I am not sure how makefiles check for updates, will the makefile with target out/minikube look for out/minikube.exe?

Yeah on Windows you'll have to build out/minikube.exe, then caching will work fine.

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

**Comment by dlorenc** [comment]
LGTM

**Pull request merged**